server/src/services/tooljet_db_operations.service.ts

```diff
@@ -74,7 +74,7 @@ export class TooljetDbOperationsService implements QueryService {
       !isEmpty(offset) && query.push(`offset=${offset}`);
     }
     const headers = { 'data-query-id': queryOptions.id, 'tj-workspace-id': queryOptions.organization_id };
-    const url = `/api/tooljet-db/proxy/${tableId}` + `?${query}`;
+    const url = `/api/tooljet-db/proxy/${tableId}` + `?${query.join('&')}`;
```

Comment:
Nit: can we only add this string `?...` when query array has something? The URL which it hits looks unpolished otherwise without any query params